innopoints/views/account.py

```diff
@@ -223,3 +247,37 @@ def change_telegram(email):
         abort(400, {'message': 'Data integrity violated.'})
 
     return NO_PAYLOAD
+
+
+@api.route('/account/notification_settings', methods=['PATCH'], defaults={'email': None})
+@api.route('/account/<email>/notification_settings', methods=['PATCH'])
+@login_required
+def change_notification_settings(email):
+    """Get the notification settings of the account.
+    If the e-mail is not passed, return own settings."""
+    if not request.is_json:
+        abort(400, {'message': 'The request should be in JSON.'})
+
+    if email is None:
+        user = current_user
+    else:
+        if not current_user.is_admin:
```

Comment:
Why should an admin be allowed to modify a user's notification settings? Users disable them for a reason.

Reply:
Same question for Telegram. Why can an admin edit a user's Telegram?

Reply:
Well, Andrey wanted ultimate powers over the website, he got them. The reason for those endpoints is so that the `/profile/<email>` page on the frontend would fully function, doesn't mean they will actually be used